diff --git a/dist/index.js b/dist/index.js
index df4c9ac74b31c91c5d65c10602873e62aadf7f30..9350f65a2a1b0cd9c57bb349da852e96d4fa70db 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -215,10 +215,10 @@ var generateElement = ({ code = "", scope = {}, enableTypeScript = true }, error
   const transformed = compose(
     addJsxConst,
     transform({ transforms: ["imports"] }),
-    wrapReturn,
     spliceJsxConst,
     trimCode,
     transform({ transforms: firstPassTransforms }),
+    wrapReturn,
     trimCode
   )(code);
   return errorBoundary_default(